client/gutenberg/extensions/markdown/markdown-editor.scss

```diff
@@ -2,7 +2,16 @@
 	opacity: 0.5;
 	pointer-events: none;
 }
-.wp-block-a8c-markdown__preview p {
+
+.wp-block-a8c-markdown__preview {
 	min-height: 1.8em;
-	white-space: pre-wrap;
+}
+
+.wp-block-a8c-markdown {
+	.wp-block-a8c-markdown__editor {
+		&:focus {
+			border-color: transparent;
+			box-shadow: 0 0 0 transparent;
```

Comment:
This would normally seem like an accessibility issue; without the border, it's hard to tell when `textarea` is active/focused.

I don't see it behaving any different from tinymce instances though so it's consistent. The only other place where I could find `textarea` was in the shortcode block, where we indeed have the accessibility-border:

<img width="676" alt="image" src="https://user-images.githubusercontent.com/87168/45351429-d5ca0380-b5be-11e8-846d-0f63a50e97a2.png">
<img width="658" alt="image" src="https://user-images.githubusercontent.com/87168/45351454-e8443d00-b5be-11e8-9bb6-4ba9c3327afe.png">

Suppose we can merge as-is and see if later core changes behaviour for tinymce instances.

Reply:
> I don't see it behaving any different from tinymce instances though so it's consistent.

Exactly the reason why I left it this way. I'm happy to update it eventually, but for now consistency with the rest of Gutenberg sounds like a good plan.

> Suppose we can merge as-is and see if later core changes behaviour for tinymce instances.

👍 💯